client_internal_test.go

```diff
@@ -24,6 +24,8 @@ func TestClient_NewRequest(t *testing.T) {
 	} else if req.URL.Path != "/v2/accounts/14579" {
 		t.Fatalf("unexpected path: %s", req.URL.Path)
 	} else if req.Method != "GET" {
+	} else if req.Header.Get("Authorization") != "Basic abc" {
```

Comment:
Woops! Fixed in [1025e1](https://github.com/blacklightcms/recurly/pull/16/commits/45f5774417cfc19c36b32e163e680758301025e1)